gapic/src/main/com/google/gapid/views/PipelineView.java

```diff
@@ -101,12 +112,17 @@ public PipelineView(Composite parent, Models models, Widgets widgets) {
     loading = LoadablePanel.create(this, widgets,
         panel -> createComposite(panel, new FillLayout(SWT.VERTICAL)));
 
-    stagesContainer = createComposite(loading.getContents(), new FillLayout());
+    stagesContainer = createComposite(loading.getContents(), new GridLayout(1, false));
+
+    RGB selectRGB = getDisplay().getSystemColor(SWT.COLOR_BLUE).getRGB();
+    RGB unselectRGB = getDisplay().getSystemColor(SWT.COLOR_LIST_BACKGROUND).getRGB();
+    hoverColor = new Color(getDisplay(), lerp(selectRGB, unselectRGB, 0.5f), (int)(0.5 * 255));
```

Comment:
Please use:
```hoverColor = new Color(getDisplay(), lerp(selectRGB, unselectRGB, 0.5f));```
We want 100% opacity here, because if the color is 50% transparent and it would work, we would see 50% of the desktop shining through. This is not intended.
If the design doc says 50% opacity, it means you should apply 50% of the foreground color over the background color. This is achieved with the 50% interpolation.